server/store_statistics.go

```diff
@@ -73,6 +74,12 @@ func (s *storeStatistics) Observe(store *core.StoreInfo) {
 	id := strconv.FormatUint(store.GetId(), 10)
 	balanceScoreGauge.WithLabelValues(s.namespace, id, "region").Set(store.RegionScore())
 	balanceScoreGauge.WithLabelValues(s.namespace, id, "leader").Set(store.LeaderScore())
+
+	storeID := fmt.Sprintf("store_%d", store.GetId())
+	storeStatusGauge.WithLabelValues(storeID, "region_size").Set(float64(store.RegionSize))
+	storeStatusGauge.WithLabelValues(storeID, "region_count").Set(float64(store.RegionCount))
+	storeStatusGauge.WithLabelValues(storeID, "leader_size").Set(float64(store.LeaderSize))
+	storeStatusGauge.WithLabelValues(storeID, "leader_count").Set(float64(store.LeaderCount))
```

Comment:
Maybe better to merge `balanceScoreGauge` with it.
BTW, we usually use only id as metrics label and config grafana to show the label as `tikv-{{id}}`.